reVX/config/dist_to_ports.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+"""
+reVX DistToPorts Configuration
+"""
+
+from reV.config.base_analysis_config import AnalysisConfig
+
+
+class DistToPortsConfig(AnalysisConfig):
+    """Config framework for distance to port calculation"""
+
+    NAME = 'meanWindDirs'
+    REQUIREMENTS = ('ports_fpath', 'excl_fpath')
+
+    def __init__(self, config):
+        """
+        Parameters
+        ----------
+        config : dict
+            Dictionary with pre-extracted config input group.
```

Comment:
Can't this also be a filepath to config?

Reply:
good question, do you know? ;-) I can look

Reply:
haha yes, str filepath to json config or pre-extracted config in dict form